pubsub/kafka/kafka.go

```diff
@@ -75,21 +97,3 @@ func (p *PubSub) Close() (err error) {
 func (p *PubSub) Features() []pubsub.Feature {
 	return nil
 }
-
-// subscribeAdapter is used to adapter pubsub.Handler to kafka.EventHandler with the same content.
-type subscribeAdapter struct {
```

Comment:
Don't remove this subscribeAdapter, you should update it. So your update can effect in all kafka components.

Reply:
Addressed